kangaroo.config.ts

```diff
@@ -1,15 +1,18 @@
 import { defineConfig } from './src/main/defineConfig';
 
 export default defineConfig({
-  appId: 'org.holochain.kangaroo-electron',
-  productName: 'Holochain Kangaroo Electron',
+  appId: 'org.holochain.kangaroo-electron.configtest',
+  productName: 'Holochain Kangaroo Electron (configtest)',
   version: '0.1.0',
   macOSCodeSigning: false,
   windowsEVCodeSigning: false,
   fallbackToIndexHtml: true,
   autoUpdates: true,
   systray: true,
   passwordMode: 'password-optional',
+  bootstrapUrl: 'https://dev-test-bootstrap2.holochain.org/',
+  signalingUrl: 'wss://dev-test-bootstrap2.holochain.org/',
```

Comment:
```suggestion
  signalUrl: 'wss://dev-test-bootstrap2.holochain.org/',
```

Reply:
There's a bunch of places where we're using "signaling" - prob easiest to do search and replace to catch them all. Would be good to be consistent with the holochain terminology.

Reply:
Fair enough. It's not really technically correct w.r.t webRTC terminology I think but better to be consistent.

Reply:
bbb7c9f16207506a14b07e42893f67cad9574127